go/cmd/dolt/commands/sql.go

```diff
@@ -1130,10 +1130,6 @@ func (se *sqlEngine) prettyPrintResults(ctx context.Context, sqlSch sql.Schema,
 
 	// Parts of the pipeline depend on the output format, such as how we print null values and whether we pad strings.
 	switch se.resultFormat {
-	case formatCsv:
```

Comment:
Was this just unnecessary? Already not printing nil values?

Reply:
`NULL`/`nil` printing is handled by the sink now. We need to eliminate this transform to be able to disambiguate between `NULL` values and empty strings